pkg/config/validation/validation.go

```diff
@@ -1724,6 +1728,39 @@ func validateTrustDomainConfig(config *meshconfig.MeshConfig) (errs error) {
 	return
 }
 
+func ValidateMeshTLSConfig(mesh *meshconfig.MeshConfig) (errs error) {
+	if meshMTLS := mesh.MeshMTLS; meshMTLS != nil {
+		if meshMTLS.EcdhCurves != nil {
+			errs = multierror.Append(errs, errors.New("mesh tls does not support ecdh curves configuration"))
+		}
+	}
+	return errs
+}
+
+func ValidateMeshExternalTLSConfig(mesh *meshconfig.MeshConfig) (v Validation) {
+	invalidEcdhCurves := sets.New[string]()
+	validEcdhCurves := sets.New[string]()
+	duplicateEcdhCurves := sets.New[string]()
+	if meshExternalTLS := mesh.MeshExternal_TLS; meshExternalTLS != nil {
+		for _, cs := range meshExternalTLS.EcdhCurves {
+			if !security.IsValidECDHCurve(cs) {
+				invalidEcdhCurves.Insert(cs)
+			} else if validEcdhCurves.InsertContains(cs) {
+				duplicateEcdhCurves.Insert(cs)
+			}
+		}
+	}
+
+	if len(invalidEcdhCurves) > 0 {
+		v = appendWarningf(v, "detected invalid ecdh curves: %v", sets.SortedList(invalidEcdhCurves))
```

Comment:
```suggestion
		v = appendWarningf(v, "detected unrecognized ECDH curves: %v", sets.SortedList(invalidEcdhCurves))
```

This wording might be less confusing, since we don't actually know if they're invalid, only Envoy knows that, they just don't match our hardcoded validation list.

Reply:
done

Reply:
Thanks!